docs/kind.md

```diff
@@ -102,7 +101,6 @@ usage: kind.sh [[[-cf |--config-file <file>] [-kt|keep-taint] [-ha|--ha-enabled]
 -ha  | --ha-enabled                Enable high availability. DEFAULT: HA Disabled.
 -ho  | --hybrid-enabled            Enable hybrid overlay. DEFAULT: Disabled.
 -ds  | --disable-snat-multiple-gws Disable SNAT for multiple gws. DEFAULT: Disabled.
--dp  | --disable-pkt-mtu-check     Disable checking packet size greater than MTU. Default: Disabled
```

Comment:
Should this be part of the previous "Handling packet sizes greater than pod MTU" patch?

Reply:
This change was introduced in a separate commit (9137c9a) which aims to improve kind documentation, not mtu specific commit.